inclusive-design/guide
# feat: add basic home page structure and CMS previews

## Description

feat: add content block editor component
feat: add call to action editor component and shortcode
feat: add home page preview
chore(deps): replace Netlify CMS with Decap CMS
feat: add serverless previews for Decap CMS

## Steps to test

1. Log in to the admin interface.

**Expected behavior:** See previews in the editing interface for pages.

## Additional information

Not applicable.

## Related issues

Not applicable.
